test/code.jl

```diff
@@ -101,18 +101,18 @@ nanmath_st.rewrites[:nanmath] = true
 
     @test toexpr(a^2) == :($(^)(a, 2))
     @test toexpr(a^2, nanmath_st) == :($(NaNMath.pow)(a, 2))
-    @test toexpr(NaNMath.pow(a, 2)) == :($(NaNMath.pow)(a, 2))
+    @test toexpr(NaNMath.pow(a, 2)) == :($(^)(a, 2))
     @test toexpr(NaNMath.pow(a, 2), nanmath_st) == :($(NaNMath.pow)(a, 2))
 
     @test toexpr(a^-1) == :($(/)(1, a))
     @test toexpr(a^-1, nanmath_st) == :($(/)(1, a))
-    @test toexpr(NaNMath.pow(a, -1)) == :($(NaNMath.pow)(a, -1))
-    @test toexpr(NaNMath.pow(a, -1), nanmath_st) == :($(NaNMath.pow)(a, -1))
+    @test toexpr(NaNMath.pow(a, -1)) == :($(inv)(a))
+    @test toexpr(NaNMath.pow(a, -1), nanmath_st) == :($(inv)(a))
 
     @test toexpr(a^-2) == :($(/)(1, $(^)(a, 2)))
     @test toexpr(a^-2, nanmath_st) == :($(/)(1, $(NaNMath.pow)(a, 2)))
-    @test toexpr(NaNMath.pow(a, -2)) == :($(NaNMath.pow)(a, -2))
-    @test toexpr(NaNMath.pow(a, -2), nanmath_st) == :($(NaNMath.pow)(a, -2))
+    @test toexpr(NaNMath.pow(a, -2)) == :($(NaNMath.pow)($(inv)(a), 2))
```

Comment:
Actually, why is this and the line below not
```suggestion
    @test toexpr(NaNMath.pow(a, -2)) == :($(^)($(inv)(a), 2))
```